### DISCUSSION TITLE
Choosing the size of the text

### DISCUSSION THREAD

**Discussion by LiveManRU** [outline/outline#4746] [Ideas]
A little more freedom in choosing the size of the text, it really hurts when the change from H1 to H3 can only be made by completely deleting the block
![Little freedom](https://i.ibb.co/tHDCY8j/2023-01-20-17-39-07.png)

**Comment by Rumcajsev** [comment]
While I agree that it could be a bit more useful, with H3 being shown as well, you can always select the text and use a shortcut (⇧ + ^ + 1/2/3) to quickly change the selected text to one of the headings.

**Reply by tommoor** [reply]
Thanks! As we get more and more non-tech folks using Outline we definiitely need to improve the UI here to show all available options. We have leant on Markdown perhaps too much in the past…

As such, you can also change the heading size by typing a certain number of `#` characters at the front of the text followed by a space.
